src/app/components/Layout.js

```diff
@@ -14,8 +14,9 @@ class Layout extends React.Component {
     this.signOut = this.signOut.bind(this);
     this.state = {
       name: '',
-      sort: [],
+      sortArr: [],
```

Comment:
I believe there can be a better descriptive name here like you did for the other state properties. Also, try to AVOID adding datatypes to names of your variables e.g sortedResult instead of sortArray ;)